File: src/Python/easy/math/problem2651.py
class Solution:
    def findDelayedArrivalTime(self, arrivalTime: int, delayedTime: int) -> int:
        time = arrivalTime + delayedTime
        if time >= 24:
            return time - 24 
        else:
            return time

arrivalTime = 15; delayedTime = 5 

solution = Solution()
solution.findDelayedArrivalTime(arrivalTime, delayedTime)

File: src/JavaScript/easy/math/problem2651.js
/**
 * @param {number} arrivalTime
 * @param {number} delayedTime
 * @return {number}
 */

var findDelayedArrivalTime = function(arrivalTime, delayedTime) {
    let time = arrivalTime + delayedTime
    if(time >= 24){
        return time - 24
    } else {
        return time
    }
};

let arrivalTime = 15, delayedTime = 5 

findDelayedArrivalTime(arrivalTime, delayedTime)